### PR TITLE
add mybinder for examples. update license. add authors file.

### PR DESCRIPTION
**Pull request by amandadumi**
Let me know if the format of the authors file is okay! Additionally, I have the binder generate within the examples folder, so it might make sense to add a section of the README that describes available examples and move the badge there.

### CODE REVIEW

**Comment by dlf57** [comment]
I just tried running a binder and everything worked with the exception of pandas. Do we need to add some sort of requirements file or environment file in order to get this to work?

**Comment by amandadumi** [comment]
we do! I'll add that, I forgot it wouldn't take it from the requirements already there. my bad.

**Comment by amandadumi** [comment]
You're right at that pandas is missing. it's because binder sources from the requirements.txt file in the base repo. Since we only need pandas for these examples and not the reps, it's not an actual dependency. I don't see an easy way to list it only as a dependency for the notebooks, but i'm looking further into it.

I will fix these commits, but i need to push to my branch to test binder right now

**Comment by dlf57** [comment]
Can we do something with an enviroment.yml as we did for QMMM?

File: AUTHORS.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+=================================
+Contributors to the chemreps code
+=================================
+
+In alphabetical order:
+Amanda Dumi
+Dakota Flomsbee
```
Comment (dlf57) [inline AUTHORS.md]: Small nitpick but that should be Folmsbee

Comment (dlf57) [reply AUTHORS.md]: Also, the markdown is not formatting correctly

Comment (amandadumi) [reply AUTHORS.md]: haha this it not a nitpick! this is your name!  butit is fixed now, and i changed are names to a bulleted list.

**Comment by amandadumi** [comment]
So as of now, I actually think the best option is to have a requirements.txt that exists only inside of a binder directory that can install binder from pip and then pandas.  Mybinder will default to look in the binder folder,  but when  the binder folder exists, it no longer will install the setup.py by default. This necessitates the install from pip.  What do you think about this?

Did we reset up the pypi to reflect the organization switch? This links (https://pypi.org/project/chemreps/) homepage points back to @dlf57 repository and the badges seem off.

**Comment by dlf57** [comment]
What if we did a `pip install git+https://github.com/chemreps/chemreps`? I think it would be better to point here as it will ensure that it is always grabbing the latest reps even if we have yet to update pypi. Also, yes we do need to update pypi.

**Comment by dlf57** [comment]
Also, I had missed your comment at the top about posting the binder badge in the readme of the examples. Do you think it would be good to have it in both readmes? That way if someone is just checking the examples out they can click it from there instead of going all the way back but also keep it on the main readme as a quick way if people know they just want to start looking at the examples?

**Comment by amandadumi** [comment]
> Edit2: I see you have a requirements.txt instead of the postBuild script. What if we put `git+git://github.com/chemreps/chemreps@master` in the script as they talk about [here](https://stackoverflow.com/questions/16584552/how-to-state-in-requirements-txt-a-direct-github-source)

this is exactly what I was looking for. I didn't love installing from pip as updates between versions may include new tests. this is much nicer. Thanks!



> Do you think it would be good to have it in both readmes?

Absolutely. This is the kind of clearness I was hoping for, slipped my mind we had a README in examples already! I have made those changes.

**Comment by shivupa** [comment]
Thanks for figuring this out guys. It's nice to see this is possible.

**Pull request merged**